ui/app/controllers/vault/cluster/dashboard.js

```diff
@@ -12,7 +12,8 @@ export default class DashboardController extends Controller {
   @tracked replicationUpdatedAt = timestamp.now().toISOString();
 
   @action
-  refreshModel() {
+  refreshModel(e) {
```

Comment:
are these tied to form submit actions now?

Reply:
They are not tied to form-submit actions! They're passed down to the child component. Are prevent defaults usually for forms only?

Reply:
It's to prevent the default behavior for that event. So we often put them in methods handling a `submit` because the default behavior is to refresh the whole page. [ref](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)

It doesn't seem like an event is passed here we need to intercept? But I could be wrong!

Reply:
I noticed a weird refresh happening without the `preventDefault` I _think_ since these are `onClick` events there is an event passed into the action when it's triggered. I put debuggers in the actions and did see that are there are events!